sdk/python/packages/flet/src/flet/cli/commands/pack.py

```diff
@@ -116,15 +116,36 @@ def add_arguments(self, parser: argparse.ArgumentParser) -> None:
         )
 
     def handle(self, options: argparse.Namespace) -> None:
+        is_dir_not_empty = lambda dir: os.path.isdir(dir) and len(os.listdir(dir)) != 0
+
         # delete "build" directory
         build_dir = os.path.join(os.getcwd(), "build")
-        if os.path.exists(build_dir):
-            shutil.rmtree(build_dir, ignore_errors=True)
+        if is_dir_not_empty(build_dir):
+            ask_to_delete = input('Do you want to delete "build" directory? (y/n) ')
+            if not ask_to_delete.lower() == "n":
+                shutil.rmtree(build_dir, ignore_errors=True)
+            else:
+                print('Failing... "build" directory must be empty to proceed.')
+                exit(1)
+
+        # delete "dist" directory or DISTPATH directory
+        # if --distpath cli option is specified
+        if options.distpath:
+            dist_dir = os.path.join(os.getcwd(), options.distpath)
+        else:
+            dist_dir = os.path.join(os.getcwd(), "dist")
 
-        # delete "dist" directory
-        dist_dir = os.path.join(os.getcwd(), "dist")
-        if os.path.exists(dist_dir):
-            shutil.rmtree(dist_dir, ignore_errors=True)
+        if is_dir_not_empty(dist_dir):
+            ask_to_delete = input(
+                f'Do you want to delete "{os.path.basename(dist_dir)}" directory? (y/n) '
+            )
+            if not ask_to_delete.lower() == "n":
```

Comment:
When we introduce interactive prompts to a CLI I believe we should add some sort of `--force` or `--yes` option, so the question is skipped in non-interactive mode, right?

Reply:
> I believe we should add some sort of --force or --yes

Working on the issue 👍

Reply:
Made the necessary changes @FeodorFitsner 👍